pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -115,6 +117,9 @@ stdenv.mkDerivation ({
     patchShebangs make
 
     ${postPatch}
+  '' + optionalString patchShell ''
```

```suggestion
  '' + optionalString patchShell ''
```
I think you should test version here replacing `patchShell`. using `lib.versionOlder`. Does it make sense?